src/plugins/data_table/data_table.js

```diff
@@ -433,6 +433,16 @@ var LocalDataTable = (function() {
         var title = this.textContent || this.innerText;
         var col = cg.columnConfigByTitle.attributes[title];
 
+        // Try to grab it by attribute, if possible
+        var matches = this.className.match(/(?:^|\s)column-(.*?)(?:\s|$)/);
+        if (matches && matches[1]) {
+          cg.columnsConfig.forEach(function(currentObject){
+            if (currentObject.attr && Backdraft.Utils.toCSSClass(currentObject.attr).replace("column-","") === matches[1]) {
+              col = currentObject;
+            }
+          })
+        }
```

Comment:
wouldn't it be easier to follow the logic if the below line was the `else` block here? (and not always executed on L434 since we may just overwrite it..)

```
col = cg.columnConfigByTitle.attributes[title];
```